comfy/model_management.py

```diff
@@ -254,7 +254,24 @@ def get_torch_device_name(device):
         return "CUDA {}: {}".format(device, torch.cuda.get_device_name(device))
 
 try:
-    print("Device:", get_torch_device_name(get_torch_device()))
+    torch_device_name = get_torch_device_name(get_torch_device())
+
+    if "[ZLUDA]" in torch_device_name:
+        print("Detected ZLUDA, support for it is experimental and comfy may not work properly.")
+
+        if torch.backends.cudnn.enabled:
+            torch.backends.cudnn.enabled = False
+            print("Disabling cuDNN because ZLUDA does currently not support it.")
+
+        torch.backends.cuda.enable_flash_sdp(True)
+        torch.backends.cuda.enable_math_sdp(False)
+        torch.backends.cuda.enable_mem_efficient_sdp(False)
+
+        if ENABLE_PYTORCH_ATTENTION:
+            print("Disabling pytorch cross attention because ZLUDA does currently not support it.")
+            ENABLE_PYTORCH_ATTENTION = False
```

Comment:
Why are you disabling pytorch attention after doing torch.backends.cuda.enable_flash_sdp(True) above?

Reply:
That's from vlad, they can all 3 be disabled atm i guess cause using sdp causes a driver timeout
<https://github.com/vladmandic/automatic/blob/2785af1bc4bff2041d09f5cd9ff7877709c75761/modules/zluda.py>

Reply:
Oh I see he only enables the math one:  torch.backends.cuda.enable_math_sdp(True)
And disables the others that makes more sense.

Reply:
Does indeed, dont know how I didn't spot that I switched that up